### PR TITLE
Update nslookup.md

### PR DESCRIPTION
**Pull request by ciis0**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@ciis0 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/windows-commands/nslookup.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ If you need to look up more than one piece of data, you can use interactive mode
 
 - Exit, by typing **exit**.
 
-- Treat a built-in command as a computer name, by preceding it with the escape character (\). An unrecognized command is interpreted as a computer name.
+- Treat a built-in command as a computer name, by preceding it with the escape character (\\). An unrecognized command is interpreted as a computer name.
```
Comment (illfated) [inline WindowsServerDocs/administration/windows-commands/nslookup.md:28]: Good improvement, this update & correction is required. Could we also take the time to add 1 single blank space before & after the backslash(es) to make the combination of parentheses and backslash look less like an integrated symbol?
```suggestion
- Treat a built-in command as a computer name, by preceding it with the escape character ( \\ ). An unrecognized command is interpreted as a computer name.
```

**Review by illfated — APPROVED** [review]
Regular source view:
- https://github.com/MicrosoftDocs/windowsserverdocs/pull/5585/files

Rich Diff view, to see for yourself why this added escape character is needed:
- https://github.com/MicrosoftDocs/windowsserverdocs/pull/5585/files?short_path=da537a9#diff-da537a977b8437c9dfeb2aa6a422acd4a44c8a414d1f32a0ab1f2ce9d54b981d

File: WindowsServerDocs/administration/windows-commands/nslookup.md (hunk 1 above)
Comment (ciis0) [reply WindowsServerDocs/administration/windows-commands/nslookup.md:28]: what about "(`\`)"? :)

**Comment by PRMerger18** [comment]
@ciis0 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**